entries/workflow.qmd

```diff
@@ -8,45 +8,164 @@ date: last-modified
 When adding or modifying content on either the website or the product
 itself, we follow the "branch-pull request" workflow. This is described
 in more detail in the [GitHub
-Flow](https://docs.github.com/en/get-started/quickstart/github-flow)
+flow](https://docs.github.com/en/get-started/quickstart/github-flow)
 page.
 
+When creating branches and pull requests, follow these general
+guidelines:
+
+-   Create branches following the [Conventional
+    Branches](https://github.com/pshaddel/conventional-branch) approach
+    by using the VSCode Extension, that is also in our (automatically)
+    recommended extension list (see tip below).
+-   When creating branches and pull requests, keep them as focused and
+    specific to the issue as is reasonable and limit how many changes
+    are made in that Pull Requst. The fewer changes made in a pull
+    request, the easier and faster it is for reviewers to look it over,
+    make suggestions, and merge it in.
+-   After creating a pull request, add it to the relevant GitHub Project
+    since this is where we keep an overview of what each of us is
+    currently working on.
+-   As you work on your branch and it starts growing too large in scope
+    and size, strongly consider creating "stacked pull requests" by
+    making a new branch on your current working branch and submit a new
+    pull request on top of the parent pull request branch.
+-   In the pull request description, try to explain *why* you made the
+    changes in the pull request, rather than the *what*.
+
+::: callout-tip
+### Creating conventional branches using the VS Code extension
+
+#### What are conventional branches?
+
+For naming branches, we follow naming scheme called **conventional
+branches**. Conventional branches are a way of naming branches that are
+easy to read and understand. This naming follow a specific format that
+includes a type and a branch description. Specifically, the format of a
+conventional branch name is `<type>/<branch>`, where `<type>` is the
+type of the branch and `<branch>` is a short description of the change
+that will be made. The description should be written in imperative mood
+using kebab-case.
```

Comment:
Maybe a bracket at the end (eg. whitespace-replaced-with-dashes)?

Reply:
Good idea. I have added some examples :)